dbms/src/Common/getNumberOfPhysicalCPUCores.cpp

```diff
@@ -1,5 +1,7 @@
 #include <Common/getNumberOfPhysicalCPUCores.h>
 #include <thread>
+#include <fstream>
+#include <math.h>
```

Comment:
No need for `round`.
You may round up, like this: `(allocated_cpus_share_int + 999) / 1000`

Reply:
Fixed, thanks